azext_edge/edge/providers/orchestration/instances.py

```diff
@@ -0,0 +1,75 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from typing import List, Optional
+
+from knack.log import get_logger
+from rich import print
+
+from ...util.az_client import get_iotops_mgmt_client, parse_resource_id
+from ...util.queryable import Queryable
+
+logger = get_logger(__name__)
+
+
+QUERIES = {
```

Comment:
are we making this a dict with one entry?

Reply:
It's a dict if more entries will be added later, similar to pattern in connected_cluster

Reply:
ok got it, thanks!

Reply:
We can refactor later if no new queries need to be made on behalf of instances.